File: WinForms/CS/HtmlMainDemo/Modules/Form/DemoDirectXForm.cs
using DevExpress.Utils;
using DevExpress.Utils.Html;
using DevExpress.XtraEditors;

namespace DevExpress.HTML.Demos {
    public partial class DemoDirectXForm : DirectXForm {
        public DemoDirectXForm() {
            InitializeComponent();
            HtmlElementMouseDown += DemoDirectXForm_HtmlElementMouseDown;
        }

        void DemoDirectXForm_HtmlElementMouseDown(object sender, DxHtmlElementMouseEventArgs e) {
            var args = e.MouseArgs as DXMouseEventArgs;
            if(e.Element == null || args == null)
                return;
            var id = e.Element.Id;
            if(e.HasClassName("addbutton")) {
                args.Handled = true;
                return;
            }
            if(id == "loginbutton") {
                args.Handled = true;
                return;
            }
            if(id == "infobutton") {
                args.Handled = true;
                return;
            }
        }

        protected override void OnShown(System.EventArgs e) {
            base.OnShown(e);
            if(!HtmlTemplate.Template.Contains(searchEdit.Name))
                searchEdit.Visible = false;
        }
    }
}

File: WinForms/VB/HtmlMainDemo/Modules/Form/DemoDirectXForm.vb
Imports DevExpress.Utils
Imports DevExpress.Utils.Html
Imports DevExpress.XtraEditors

Namespace DevExpress.HTML.Demos

    Public Partial Class DemoDirectXForm
        Inherits DirectXForm

        Public Sub New()
            InitializeComponent()
            AddHandler HtmlElementMouseDown, AddressOf DemoDirectXForm_HtmlElementMouseDown
        End Sub

        Private Sub DemoDirectXForm_HtmlElementMouseDown(ByVal sender As Object, ByVal e As DxHtmlElementMouseEventArgs)
            Dim args = TryCast(e.MouseArgs, DXMouseEventArgs)
            If e.Element Is Nothing OrElse args Is Nothing Then Return
            Dim id = e.Element.Id
            If e.HasClassName("addbutton") Then
                args.Handled = True
                Return
            End If

            If Equals(id, "loginbutton") Then
                args.Handled = True
                Return
            End If

            If Equals(id, "infobutton") Then
                args.Handled = True
                Return
            End If
        End Sub

        Protected Overrides Sub OnShown(ByVal e As System.EventArgs)
            MyBase.OnShown(e)
            If Not HtmlTemplate.Template.Contains(searchEdit.Name) Then searchEdit.Visible = False
        End Sub
    End Class
End Namespace
